### ISSUE TITLE
Lines? [feature request]

### ISSUE THREAD

**Issue by SpooikypokDev** [dalboris/vpaint#68]
A cool thing would be to be able to create straight lines by pressing the shift key while clicking with the pen, so a line would be drawn between the last location of the pen and the current location of the pen.

**Comment by dalboris** [comment]
Yes, I totally agree. In fact, I did implement this exact feature in a very early version of VPaint (before animation features were present), but I had to remove it in order to be able to make more progress on the animation front. It should be back in a "near" (1-2 years) future. :)

**Comment by SpooikypokDev** [comment]
Should this issue be considered as closed or should it be left open to be closed when the straight lines feature is implemented?

**Comment by dalboris** [comment]
We'll leave it open until it's implemented :)
